etna/core/models/basepage.py

```diff
@@ -145,6 +145,19 @@ def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
         data.update(customDimension3=self._meta.verbose_name)
         return data
 
+    @property
+    def privacy(self):
+        privacy = [r.restriction_type for r in self.get_view_restrictions()]
+        if privacy:
+            return privacy[0]
+        return "public"
+
+    meta_fields = [
+        "privacy",
+        "latest_revision_created_at",
```

Comment:
Nice work AJ - just one question, why do you want `latest_revision_created_at`?

Reply:
Sorry, I was trying to get `last_published_at`. Updated now.